apps/passport-server/src/apis/zuconnect/zuconnectTripshaAPI.ts

```diff
@@ -8,21 +8,27 @@ import { logger } from "../../util/logger";
  */
 const TRIPSHA_TICKET_TYPES = [
   "ZuConnect Resident Pass",
+  "1st Week Pass",
+  "ZuConnect Scholarship",
```

Comment:
We don't yet have a confirmed type for day passes. I will try to lock these down so that we don't have to keep adding more.